RecoPixelVertexing/PixelTrackFitting/python/PixelTracks_cff.py

```diff
@@ -83,8 +84,16 @@
 # Use Riemann fit and substitute previous Fitter producer with the Riemann one
 from Configuration.ProcessModifiers.riemannFit_cff import riemannFit
 from Configuration.ProcessModifiers.riemannFitGPU_cff import riemannFitGPU
-riemannFit.toModify(pixelTracks, Fitter = "pixelFitterByRiemannParaboloid")
```

Comment:
Ok! Since I'm not sure about how to change it (the file was changed by Felice...), please tell me if replacing
~~~
# Use Riemann fit and substitute previous Fitter producer with the Riemann one
from Configuration.ProcessModifiers.riemannFit_cff import riemannFit
from Configuration.ProcessModifiers.riemannFitGPU_cff import riemannFitGPU

# Comment/uncomment to switch between broken line and Riemann fit (and do the same below)
riemannFit.toModify(pixelTracks, Fitter = "pixelFitterByBrokenLine")
#riemannFit.toModify(pixelTracks, Fitter = "pixelFitterByRiemannParaboloid")

riemannFitGPU.toModify(pixelTracks, runOnGPU = True)
_pixelTracksSequence_riemannFit = pixelTracksSequence.copy()

# Comment/uncomment to switch between broken line and Riemann fit (and do the same above)
_pixelTracksSequence_riemannFit.replace(pixelFitterByHelixProjections, pixelFitterByBrokenLine)
#_pixelTracksSequence_riemannFit.replace(pixelFitterByHelixProjections, pixelFitterByRiemannParaboloid)

riemannFit.toReplaceWith(pixelTracksSequence, _pixelTracksSequence_riemannFit)
~~~
with
~~~
# Use Riemann/BrokenLine fit and substitute previous Fitter producer with the Riemann/BrokenLine one
from Configuration.ProcessModifiers.riemannFit_cff import riemannFit
from Configuration.ProcessModifiers.riemannFitGPU_cff import riemannFitGPU
from Configuration.ProcessModifiers.brokenLine_cff import brokenLine
from Configuration.ProcessModifiers.brokenLineGPU_cff import brokenLineGPU

riemannFit.toModify(pixelTracks, Fitter = "pixelFitterByRiemannParaboloid")
brokenLine.toModify(pixelTracks, Fitter = "pixelFitterByBrokenLine")

riemannFitGPU.toModify(pixelTracks, runOnGPU = True)
_pixelTracksSequence_riemannFit = pixelTracksSequence.copy()
brokenLineGPU.toModify(pixelTracks, runOnGPU = True)
_pixelTracksSequence_brokenLine = pixelTracksSequence.copy()

_pixelTracksSequence_riemannFit.replace(pixelFitterByHelixProjections, pixelFitterByRiemannParaboloid)
_pixelTracksSequence_brokenLine.replace(pixelFitterByHelixProjections, pixelFitterByBrokenLine)

# Comment/uncomment to choose the fit to be run
#riemannFit.toReplaceWith(pixelTracksSequence, _pixelTracksSequence_riemannFit)
brokenLine.toReplaceWith(pixelTracksSequence, _pixelTracksSequence_brokenLine)
~~~
would be ok. Thanks!

Reply:
it looks reasonable, except I guess
  - the last `#riemannFit.toReplaceWith(...)` should not be commented
  - you will need to create the files `Configuration/ProcessModifiers/python/brokenLine_cff.py` and `Configuration/ProcessModifiers/python/brokenLineGPU_cff.py` , which you can copyand adapt from their riemannFit variants